### PR TITLE
Add override troubleshooting in README

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Ref: https://issues.redhat.com/browse/ACM-9031

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,13 @@ ManagedClusterAddOn resource. This will enable changes to the ManifestWork on th
 persist, but direct changes to resources on a managed cluster will still be reverted to match the
 ManifestWork.
 
+### Image Override Troubleshooting
+If there is a trouble to override an image in ACM(ex: Daily cluster), `klusterletaddonconfig-pause` annotation is required to override `addon.open-cluster-management.io/values` in `ManagedClusterAddon`. 
+```shell
+oc annotate klusterletaddonconfig -n ${CLUSTER_NAME} ${CLUSTER_NAME} klusterletaddonconfig-pause=true --overwrite=true
+```
+
+
```
Comment (dhaiducek) [inline README.md:142]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -133,6 +133,13 @@ ManagedClusterAddOn resource. This will enable changes to the ManifestWork on th
 persist, but direct changes to resources on a managed cluster will still be reverted to match the
 ManifestWork.
 
+### Image Override Troubleshooting
+If there is a trouble to override an image in ACM(ex: Daily cluster), `klusterletaddonconfig-pause` annotation is required to override `addon.open-cluster-management.io/values` in `ManagedClusterAddon`.
```
Comment (dhaiducek) [inline README.md:137]:
```suggestion
### Image Override Troubleshooting

If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.

```

Comment (yiraeChristineKim) [reply README.md:137]: Thanks!

File: README.md [hunk 3]
```diff
@@ -133,6 +133,12 @@ ManagedClusterAddOn resource. This will enable changes to the ManifestWork on th
 persist, but direct changes to resources on a managed cluster will still be reverted to match the
 ManifestWork.
 
+### Image Override Troubleshooting
+If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.
```
Comment (dhaiducek) [inline README.md:137]: Could you also add newlines before and after the paragraph and add some line wrapping? It matches formatting elsewhere here. (And I think VSCode might also do this for you if you use the Prettier extension.)
```suggestion
### Image Override Troubleshooting

If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.

```

Comment (yiraeChristineKim) [reply README.md:137]: Prettier for readme?

Comment (dhaiducek) [reply README.md]: Yes--Prettier for Markdown, specifically.

File: README.md [hunk 4]
```diff
@@ -133,6 +133,14 @@ ManagedClusterAddOn resource. This will enable changes to the ManifestWork on th
 persist, but direct changes to resources on a managed cluster will still be reverted to match the
 ManifestWork.
 
+### Image Override Troubleshooting
+
+If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.
```
Comment (dhaiducek) [inline README.md:138]: Sorry, one final thing is line wrapping so this isn't a super long line:
```suggestion
If there is trouble overriding an image or other configurations in ACM, the
`klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the
`addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -133,6 +133,15 @@ ManagedClusterAddOn resource. This will enable changes to the ManifestWork on th
 persist, but direct changes to resources on a managed cluster will still be reverted to match the
 ManifestWork.
 
+### Image Override Troubleshooting
+
+If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the
```
Comment (dhaiducek) [inline README.md:138]: One more line break and it's good to go! 😬 

```suggestion
If there is trouble overriding an image or other configurations in ACM, the 
`klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the
```

Comment (yiraeChristineKim) [reply README.md]: Is it correct?

**Review by dhaiducek — APPROVED** [review]
LGTM! Thanks for updating it!

**Pull request merged**